local/bin/py/build/update_pre_build.py

```diff
@@ -44,9 +44,13 @@ def __init__(self, tempdir):
             join(self.tempdir, "extracted") + sep
         )
 
+        # Should match directory name in integrations_data/extracted
         self.apw_integrations = [
             'ably',
-            'akamai_mpulse'
+            'akamai_mpulse',
+            'avmconsulting_workday',
+            'bottomline_mainframe',
```

Comment:
It looks like on of the assets in the carousel is a vimeo video and I'm seeing some CSP errors related to it

https://docs-staging.datadoghq.com/brian.deutsch/apw-marketplace/integrations/bottomline-mainframe/
Asset: https://player.vimeo.com/video/779688046

I don't believe this is being introduced by your changes but we should create a card to get this resolved

Reply:
@brooklynebkbk weird i don't see these in my preview site, but i do see them in live.    if we can create a card to address that would be great.   thank you!